src/components/Trade/hooks/useSwapper/useSwapper.tsx

```diff
@@ -91,27 +79,6 @@ export const useSwapper = () => {
     selectBIP44ParamsByAccountId(state, buyAccountFilter),
   )
 
-  /*
```

Comment:
These removed selectors have all been moved to `src/state/zustand/swapperStore/selectors.ts`.